homework-node/helpers/writeFile.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+
+const writeFile = function write(data, pathToFile) {
```

Comment:
I think you can create a file `file.js` and move the read/write functions there.